commcare_connect/form_receiver/tests/test_receiver_integration.py

```diff
@@ -366,17 +369,10 @@ def test_auto_approve_payments_rejected_visit(
     rejected_reason.append(visit.reason)
     visit.save()
 
-    duplicate_json = deepcopy(form_json)
-    duplicate_json["id"] = str(uuid4())
-    make_request(api_client, duplicate_json, user_with_connectid_link)
-    visit = UserVisit.objects.get(xform_id=duplicate_json["id"])
-    visit.status = VisitValidationStatus.rejected
-    visit.reason = "duplicate"
```

Comment:
Is this not the behaviour any more?

Reply:
These test checks were causing tests to fail with these new additions. Adding duplicate visits causes completed works to be marked as duplicates. So checking for duplicates does not make sense in tests testing rejected behavior.